.github/src/ComputeEngine.java

```diff
@@ -1,4 +1,4 @@
-package src;
+

```

Comment:
This interface is great for future flexibility with the value being passed- It can be changed to a String type for example without limitations from the process being across multiple classes/files.